surround/runner/web/api.py

```diff
@@ -45,9 +45,14 @@ def post(self):
 
 def make_app(wrapper_object):
     predict_init_args = dict(wrapper=wrapper_object)
+    if wrapper_object.type_of_uploaded_object == AllowedTypes.FILE:
+        return tornado.web.Application([
```

Comment:
Does it this really need to be copy pasted? Just append the `upload` endpoint to the list of endpoints!

Reply:
:no_mouth:

Reply:
Fixed!